src/main/java/Alvis.java

```diff
@@ -0,0 +1,129 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Alvis {
+    private static final String FILE_PATH = "./Save.txt";
+    
+    public static void main(String[] args) throws IOException {
```

Comment:
Hmmm... 🤔 
Isn't main method too cramped right now?

Suggestion: Consider creating a different method for processing the input instead of putting everything inside main. It can cause issues when you'll be trying to further scale it and will make error handling and testing difficult. 😸 
Can create a method like startChatbot().